packages/devtools_app/lib/src/screens/deep_link_validation/select_project_view.dart

```diff
@@ -26,13 +29,20 @@ class SelectProjectView extends StatefulWidget {
 class _SelectProjectViewState extends State<SelectProjectView>
     with ProvidedControllerMixin<DeepLinksController, SelectProjectView> {
   bool _retrievingFlutterProject = false;
+  String? packageDirectoryForMainIsolate;
```

Comment:
I recommend storing this on DeepLinksController instead of in this widget state

Reply:
in fact, we should call rootPackageDirectoryForMainIsolate from the DeepLinksController so that the widget can just grab the value from the controller

Reply:
updated